### ISSUE TITLE
Images sometimes are not loading in the thread creator

### ISSUE THREAD

**Issue by RhysCommonwealth** [hicommonwealth/commonwealth#5825]
## Describe the bug
When adding an image to the thread creator window, images sometimes are not loading. 

## To Reproduce
1. Create a new thread
2. Upload an image to the thread
3. Repeat until image does not load
## Expected behavior
Images should always load in the thread creator
## Additional context
https://www.loom.com/share/6ca7cb5c65de4706800f49102646a45e

**Comment by CowMuon** [comment]
Given this appears to be a Heisenbug, a loom here would be very helpful. @RhysCommonwealth 

We'd also like to understand step 3 a little better, what is the frequency this is happening? (every n times, where n = ?) 

Finally, what happens after the image fails to load, and you re-upload it, does it just work? Or does it leave the user in a weird state?

**Comment by RhysCommonwealth** [comment]
When reproducing the bug for the ticket, I pasted an image into the thread draft twice (the loom in additional context section). The second time I pasted the image into the thread it didn't load in the text editor. The image also didn't load when I published the thread when the image didn't load in the text editor. 

I just tried to reproduce the bug in two different communities with no success. Looks like this bug is no longer on the live site. I'll close this issue.
